hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationService.java

```diff
@@ -42,6 +42,8 @@
 
     <T> T toObject(Object data);
 
+    <T> T bytesToObject(byte[] bytes, int dataOffset);
```

Comment:
I am really appreciative for this method
I was going to implement it these days :) Thanks